docs/Data correction documentary-cleared.ipynb

```diff
@@ -0,0 +1,301 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "## Data correction procedure for TOF-SANS with polarization analysis\n",
+    "\n",
+    "- Note on notation:\n",
+    "> - subscripts = discretization variable from measurement\n",
+    "> - superscripts = Names\n",
+    "> - () = continuous function of....\n",
+    "\n",
+    "| Value | Meaning | shape | How acquired? | How named in current main branch? |\n",
+    "|:--------:|:--------:|:--------:|:--------:|:--------:|\n",
+    "|  $I^{++}_{t,\\lambda}$   |  uncorrected sample data of ++ channel |  data array with columns for $I$, $\\lambda$, $t$   |  neutron data |Row 1   |\n",
```

Comment:
I don't think the notation like `$I^{++}_{t,\\lambda}$` makes sense when talking about event data. This looks like a 2-D array (for any given $t$ there can be multiple $\lambda$ and vice versa), which would be fine if we planned to histogram, but maybe need to come up with a better notation for event data? Maybe an event index?

On the other hand, maybe we can keep it, and just clarify in the beginning that the notation is for histogrammed data?

Reply:
So for that part we use histogrammed, but for sample-correction in the end the not histogrammed data?

Reply:
Let use keep histogrammed notation everywhere (at least for now). One can always think of event data as an extremely sparse histogram, so conceptually it is ok (and this document is *not* about the way Scipp handles event data).